api/preview/documentdb_types.go

```diff
@@ -67,6 +76,8 @@ type ClusterReplication struct {
 	Primary string `json:"primary"`
 	// ClusterList is the list of clusters participating in replication.
 	ClusterList []string `json:"clusterList"`
+	// Whether or not to have replicas on the primary cluster.
+	HighAvailability bool `json:"highAvailability,omitempty"`
```

Comment:
Nit: From the name, tough to understand what the comment is implying. Is it only for primary?